otm/resources/schemas/otm_schema.json

```diff
@@ -207,9 +210,9 @@
                 "properties": {
                     "id": {"type": "string"},
                     "name": {"type": "string"},
-                    "description": {"type": "string"},
+                    "description": {"type": ["string", "null"]},
                     "riskReduction": {"type": "number"},
-                    "attributes": {"type": "object"}
+                    "attributes": {"type": ["object", "null"]}
                 }
             }
         }
```

Comment:
"size" and "position" are optional for some representation types, so in those cases, they should be allowed to be null